### PR TITLE
Poshiide 22

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
Hey Greg,
I built the anyedit update site by tycho tools, but we can't specify the repository url to local path because the maven would verify url before the tycho build tools running.But I have built the update site and can you upload the files to your https://dl.bintray.com/gamerson/anyedit ?I think it will work well.
Thanks.

File: pom.xml [hunk 1]
```diff
@@ -55,6 +55,11 @@
             <layout>p2</layout>
             <url>http://oss.opensagres.fr/eclipse-wtp-xml-search/1.0.1-SNAPSHOT/</url>
         </repository>
+        <repository>
+            <id>anyedit</id>
+            <layout>p2</layout>
+            <url>https://dl.bintray.com/gamerson/anyedit/</url>
```
Comment (jtydhr88) [inline pom.xml:61]: can you upload the updatesite files to this link? thanks.

**Comment by jtydhr88** [comment]
the update site folder is \com.liferay.poshi.ide\anyedit-update-site\repository\

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Comment by gamerson** [comment]
Merged, thanks! d8e35c2...f990c50 <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Pull request closed without merging**